presto-docs/src/main/sphinx/installation.rst

```diff
@@ -2,11 +2,18 @@
 Installation
 ************
 
+A Presto server can be installed and deployed on various systems. Typically you
```

Comment:
on various systems -> on one or more systems

"various systems" seems strange -- I'm not sure what that means

Reply:
Reworded